Bottom: b8983eaf564bb7cdb8aeee9f7c364c12b63b47b2
Top:    9a7b1632d76a11f9781d6b6d7ffe0f9cf66419b6
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:58 -0500

Exec account script on exim4 container

Instead of starting a new container to add accounts,
execute script within existing container.


---
diff --git a/roles/docker-exim4/tasks/main.yml b/roles/docker-exim4/tasks/main.yml
index 5240723..e001fe9 100644
--- a/roles/docker-exim4/tasks/main.yml
+++ b/roles/docker-exim4/tasks/main.yml
@@ -122,5 +122,5 @@
 # Email accounts
 
 - name: populate email passwords
-  shell: 'docker run --rm --volumes-from {{ exim4_dv_name }} {{ exim4_image_name }}:{{ docker_image_tag }} add_account {{ item.value.user }} "{{ item.value.password }}"'
+  shell: 'docker exec {{ exim4_container_name }} /docker-entrypoint.sh add_account {{ item.value.user }} "{{ item.value.password }}"'
   with_dict: '{{ email_accounts }}'
